_couches/lafuente-hacklab.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Nino
+city: Brussels
+country: BE
+region: Anderlecht
+email: lafuente@riseup.net
+github: glycergic
+twitter: nin_o_0
+website: lafuente.space
```

Comment:
@Glycergic This line should be: 
```
website: https://lafuente.space
```

Otherwise we get this [error](https://travis-ci.org/hackercouch/hackercouch/builds/200785555):
```
- ./_site/couches/lafuente-hacklab.html
  *  internally linking to lafuente.space, which does not exist (line 130)
```